.github/workflows/conda-python-test.yaml

```diff
@@ -0,0 +1,160 @@
+on:
+    workflow_call:
+      inputs:
+        build_type:
+          required: true
+          type: string
+        branch:
+          type: string
+        date:
+          type: string
+        sha:
+          type: string
+        repo:
+          type: string
+        script:
+          type: string
+          default: "ci/test_conda_python.sh"
+        run_codecov:
+          type: boolean
+          default: false
+        matrix_filter:
+          type: string
+          default: "."
+        container-options:
+          required: false
+          type: string
+          default: "-e _NOOP"
+
+defaults:
+    run:
+      shell: bash
+
+permissions:
+    actions: read
+    checks: none
+    contents: read
+    deployments: none
+    discussions: none
+    id-token: write
+    issues: none
+    packages: read
+    pages: none
+    pull-requests: read
+    repository-projects: none
+    security-events: none
+    statuses: none
+
+jobs:
+  compute-matrix:
+    runs-on: ubuntu-latest
+    env:
+      BUILD_TYPE: ${{ inputs.build_type }}
+    outputs:
+      MATRIX: ${{ steps.compute-matrix.outputs.MATRIX }}
+    steps:
+      - name: Validate Test Type
+        run: |
+          if [[ "$BUILD_TYPE" != "pull-request" ]] && [[ "$BUILD_TYPE" != "nightly" ]]; then
```

Comment:
RAPIDS workflows have been updated to include a "build" type so that we can test branch builds. https://github.com/rapidsai/shared-workflows/pull/275